doc/classes/GraphEdit.xml

```diff
@@ -344,13 +344,14 @@
 		</signal>
 		<signal name="copy_nodes_request">
 			<description>
-				Emitted when the user presses [kbd]Ctrl + C[/kbd].
+				Emitted when this [GraphEdit] requests [GraphElement] nodes to be copied, for example by pressing [kbd]Ctrl + C[/kbd] (the [code]ui_copy[/code] action).
```

Comment:
Okay, I think this is a bit of a weird phrasing, because what it says is that a node is requesting something by pressing keys on the keyboard. 🙃 How about

```suggestion
				Emitted when a [code]ui_copy[/code] action is captured by this [GraphEdit] ([kbd]Ctrl + C[/kbd] by default). In general, you are expected to handle this signal only for selected [GraphElement]s.
```